UnityProject/Assets/Scripts/Managers/NetworkManagement/CustomNetworkManager.cs

```diff
@@ -366,12 +366,14 @@ public override void OnServerDisconnect(NetworkConnectionToClient conn)
 		//note that we can't remove authority from player owned objects, the workaround is to transfer authority to
 		//a different temporary object, remove authority from the original, and then run the normal disconnect logic
 
+
 		//transfer to a temporary object
 		GameObject disconnectedViewer = Instantiate(CustomNetworkManager.Instance.disconnectedViewerPrefab);
 		NetworkServer.ReplacePlayerForConnection(conn, disconnectedViewer, System.Guid.NewGuid());
 
 		foreach (var ownedObject in conn.clientOwnedObjects.ToArray())
 		{
+			if (disconnectedViewer == ownedObject.gameObject) continue;
```

Comment:
what does this fix?

Reply:
trying to remove authority for the cclient object , because it's removing authority for disconnectedViewer  object, when it's just been set to it

Reply:
so is this fixing the kick on join bug?

Reply:
nope, I don't think so

Reply:
just cleaning up the logs a bit